service/controller/app/v1/values/configmap_test.go

```diff
@@ -0,0 +1,358 @@
+package values
+
+import (
+	"context"
+	"reflect"
+	"strconv"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/micrologger/microloggertest"
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgofake "k8s.io/client-go/kubernetes/fake"
+)
+
+func Test_MergeConfigMapData(t *testing.T) {
```

Comment:
This basically copy pasta from the configmap desired state test. In the next PR I will simplify the UT there.